lib/std/debug/MemoryAccessor.zig

```diff
@@ -59,7 +59,15 @@ fn read(ma: *MemoryAccessor, address: usize, buf: []u8) bool {
                 switch (linux.E.init(bytes_read)) {
                     .SUCCESS => return bytes_read == buf.len,
                     .FAULT => return false,
-                    .INVAL, .SRCH => unreachable, // own pid is always valid
+                    .INVAL => unreachable, // EINVAL is returned when flags to process_vm_readv is non-zero
+                    .SRCH => {
+                        // process with pid does not exist, but it should be our pid, so we'll try again.
```

Comment:
Could we please explain an example of a case in which this can happen (post-fork()) in the comment?

Reply:
Wasn't sure how much detail to add, do you think this is enough?